pages/common/open.md

```diff
@@ -0,0 +1,15 @@
+# open
+
+> `open` can refer to multiple commands with the same name.
+
+- View documentation for the command available in macOS:
+
+`tldr open -p osx`
+
+- View documentation for the macOS command in older versions of `tldr` command-line client:
+
+`tldr open -o osx`
```

Comment:
```suggestion
```

On a second thought, we could remove this (ref. #10851) as a majority of our clients have migrated to the recent Client specification.

Reply:
Has the node.js version of the client been fixed too?

Reply:
> Has the node.js version of the client been fixed too?

Seems like the Node client hasn't been fixed yet. I will try to open a PR to update it to use `--platform` and `-p` instead of `--os` and `-o`.